src/components/Profile/components/VictoryPromise/components/GraphDisplay.tsx

```diff
@@ -1,9 +1,8 @@
 import { Polar } from 'react-chartjs-2';
 import { toTitleCase } from 'services/utils';
 import { Colors, VictoryPromiseCategory, VictoryPromiseColor } from 'services/victoryPromise';
-import { gordonColors } from 'theme';
 
-const colors = new Array<VictoryPromiseColor | typeof gordonColors.neutral.lightGray>(4);
+const colors = new Array<VictoryPromiseColor | '#dbdbda'>(4);
```

Comment:
Since this color is used in two places, it might be better to save this color to a variable in this file and then use that, up to you though.